test/e2e/lua/dynamic_configuration.go

```diff
@@ -252,3 +314,19 @@ func ensureIngress(f *framework.Framework, host string) (*extensions.Ingress, er
 		},
 	})
 }
+
+func updateConfigmap(k, v string, c kubernetes.Interface) {
```

Comment:
This method is only for the ingress-nginx nginx-configuration config map - maybe we can rename this method for clarity? Something like `updateNginxConfigMap`? WDYT?

Also since wherever this method is being used, I imagine you'd always want to reset the config map to the older value at the end of the test (relating to the earlier comment)

To avoid duplicating code, maybe you could make this method return the oldConfigMap?

Reply:
This is being addressed here: https://github.com/Shopify/ingress/pull/45